opentasks/src/main/java/org/dmfs/tasks/EditTaskFragment.java

```diff
@@ -476,7 +477,13 @@ public void onSaveInstanceState(Bundle outState)
 	@Override
 	public void onLoadFinished(Loader<Cursor> loader, Cursor cursor)
 	{
-		mTaskListAdapter.changeCursor(cursor);
+		if (cursor == null || cursor.getCount() == 0)
+		{
+			showNoListMessageAndFinish();
+			return;
+		}
+
+        mTaskListAdapter.changeCursor(cursor);
```

Comment:
Do you mean the guard with the `return`? To change that?
Or the code format style? I've applied that to these lines... but also rebased to `staging` in the meantime and I don't see those big indents now..   Can you please check again now?